### DISCUSSION TITLE
[Feature Request] The calendar `firstDayOfWeek` should respect locale

### DISCUSSION THREAD

**Discussion by tlouisse** [ing-bank/lion#610] [General]
Currently, our first day of the week is always 0 (a Sunday).

Although `firstDayOfWeek` is a configurable option on `LionCalendar`, we might want to make it locale dependent, so that it is Sunday/Monday/some other day dependent on locale.
 
See:
https://github.com/unicode-cldr/cldr-core/blob/35.0.0/supplemental/weekData.json#L60
https://github.com/tc39/ecma402/issues/6#issuecomment-114079502

In order to achieve this, we could consider a key/value store inside `localize`

**Comment by jorenbroekema** [comment]
Closing as enhancements with votes needed. Our workflow with enhancements is that we close them and have people vote on them, then we sort our enhancements by most votes to see which ones should be prioritized: https://github.com/ing-bank/lion/issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement+sort%3Areactions-%2B1-desc
